content/applications/websites/ecommerce/maximizing_revenue/reviews.rst

```diff
@@ -1,59 +1,70 @@
-===============================
-How to enable comments & rating
-===============================
+=========================
+Enable comments & ratings
+=========================
 
-Publishing and monitoring customer experience will help you gain the trust
-of new customers and better engage with your community. In 2 clicks, allow
-your customer to share their feedback!
+Properly publishing and accurately monitoring how your customers feel about the online shopping
+experience you provide is crucial for any business, especially if you want to organically gain
+the trust of your customer base and genuinely engage with your community.
```

Comment:
Hi KC, as a quick reminder, documentation is a place to inform, not to promote. For more information, check our documentation about [writing style](https://www.odoo.com/documentation/user/14.0/contributing/documentation/content_guidelines.html#writing-style).